### DISCUSSION TITLE
the options [servers] is not supported,....

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#13601] [Help]
Hi, I got this notification  in console
![image](https://user-images.githubusercontent.com/15695252/83348363-fd926980-a355-11ea-8ea8-d29c558d5a33.png)
I think it from nextjs, but I don't know what did I do wrong.
Do you have any idea for this ? It's not an error, site still run

**Comment by timneutkens** [comment]
Doesn't sound like a Next.js message.

**Reply by IRediTOTO** [reply]
:| Don't know where is it from. I just worry mongodb run on client.

**Reply by PyroStefan** [reply]
Copy the url you get from MongoDB Atlas.

If you haven't set it up already:
1. Go to your MongoDB cluster on the left ->Database Access->Add New Database User
2. Go back to Clusters>connect->Connect your application->copy the link (change <password> to the password set in step 1)

Something similar to [this](https://www.youtube.com/watch?v=OuCrHynro0w)
